overlord/snapstate/snapmgr.go

```diff
@@ -102,6 +102,11 @@ func (ss *SnapSetup) TryMode() bool {
 	return ss.Flags&TryMode != 0
 }
 
+// RevertOp returns true if the operation is a revert
+func (ss *SnapSetup) RevertOp() bool {
```

Comment:
Looks like either the revert branch leaked into this one, or this needs merging to catch up with the latest changes on revert work.